client/layout/masterbar/minimal.jsx

```diff
@@ -5,22 +5,34 @@
  */
 import React from 'react';
 import Masterbar from './masterbar';
+import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import Item from './item';
 
-const MasterbarMinimal = ( { url } ) => (
+const MasterbarMinimal = ( { url, loggedOut, children } ) => (
```

Comment:
It was being used for logged-in users in a signup A/B test flow. That test now seems to have ended, so I'm hopeful that we can move to just having one logged-out masterbar component.

@drewblaisdell: is it ok to ditch MasterbarMinimal now?